### DISCUSSION TITLE
（ご協力のお願い）モバイルの閲覧時「ページの画面下に出るポップアップが画面外にズレてしまう」現象が起きている方

### DISCUSSION THREAD

**Discussion by tanaka-asami415** [increments/qiita-discussions#801] [アナウンス]
モバイルでの閲覧時（未ログイン時）に「記事ページの画面下に出るポップアップが画面外にズレていて閉じることができない」
というご報告をいただきました。

下記画像のようなポップアップが画面の右外にズレて「また後で」が押せず、閉じる事ができなくなっているようです。
![popup (1)](https://github.com/increments/qiita-discussions/assets/103916185/a80efb94-8755-4ba6-9d49-a56981530eba)

修正を行うために確認を行ったのですが、弊社の環境では再現ができなかったため、
もしも同様の現象が起きているという方がいらっしゃいましたら、デバイス等の情報を教えていただけないでしょうか🙇

こちらのDiscussionsへの回答に不都合がある場合は、[サポートフォーム](https://support.qiita.com/hc/ja/requests/new) までご連絡いただけると幸いです。

### 未ログイン時にポップアップが表示されるページ
- TOPページ（プッシュ通知のポップアップ）
- 記事ページ（ログイン案内のポップアップ）

### 教えていただきたい内容
- デバイス
- OSのバージョン
- ブラウザの種類とバージョン

**Comment by yoshinoyuya** [comment]
自身の環境では発生していませんが、ブラウザじゃない可能性もありそうですね
以前、ブラウザではなく、アプリで見ている方も居たので
https://github.com/increments/qiita-discussions/discussions/305#discussioncomment-4163984

**Reply by tsubolabo** [reply]
ご指名に与かりましたので、画像を添付します。
「また後で」は表示されませんが、パソコン版のように それをクリックしなければ先に進めないということはなく、そのままでもスクロールはできるし、「トレンド」の記事のタイトルをクリックすれば、記事の内容は普通に見えます。特に困っていることはありません（最近はパソコンでの閲覧がメインなこともあり） 。

デバイス　XPERIA XZ SO-01J
OSのバージョン　Android 7.0
ブラウザの種類とバージョン　---
アプリ　Qiita バージョン1

![Screenshot_20240524-164930](https://github.com/increments/qiita-discussions/assets/42261605/73b08e21-50ae-44a2-a3c1-42b85a946ac7)

**Reply by tanaka-asami415** [reply]
@yoshinoyuya 
ありがとうございます！

@tsubolabo 
ご報告ありがとうございます！

今回、tsubolaboさんに添付いただいた画像は、トレンドページに表示されているものかと存じます。
こちらは、ポップアップではなく「また後で」は表示されないのは仕様です🙇

下記コメントでいただいた内容については、
サービスを維持するために、できるだけログインして使っていただきたいという思いがあり
現状のような形になっていますが、適宜見直しは行っていきたいと思います。

**Comment by tsubolabo** [comment]
いままで気になりながらも黙っていましたが、ちょうど良い機会なので感想を述べさせてください。一個人の単なる感想です。回答は要りません。

**このポップアップは倫理的に好きではありません。**

　「ログイン」「新規登録」「また後で」のそれぞれのボタンの形式が異なっています。特に、「また後で」はボタンには見えません。「新規登録」へのクリックを期待していることは痛いほど分かりますが、某通販サイトの高級会員登録への誤クリックを誘う非倫理的な画面と同様の不快さを感じます。
